config/locales/en.yml

```diff
@@ -69,6 +69,8 @@ en:
     logout: Logout
     manage: Manage %{model}
     my_tab: My %{model}
+    payment_sources: My Payment Sources
```

Comment:
Can we clean up the use of this vs `t_my(Account)` in `LinkCollection#accounts`? Haven't tried it out, but they seem like they should be the same translation.